SR2019/2019-05-15-kit-chase-round-1.md

```diff
@@ -0,0 +1,28 @@
+---
+to: Teams with outstanding kit
+subject: Outstanding kit
+---
+
+Hello!
+
+According to our records, you still have a kit from this years competition. As the competition has ended, we would like this back!
+
+Please securely pack the kit and ship it to the address listed on [our website](https://studentrobotics.org/contact/). Once you have dispatched it, please [let us know](mailto:teams@studentrobotics.org).
+
+## How to ship your kit
+
+- Kits should be shipped in the white 18l Really Useful Box (RUB). For extra protection, we recommend wrapping this in paper / cling film during shipping.
```

Comment:
nit: maybe clarify that we mean packing paper? (I don't know what the proper term is, but I have visions of them arriving wrapped in print quality paper)